config/clusters/nasa-veda/common.values.yaml

```diff
@@ -117,6 +149,38 @@ basehub:
                         mountPath: /home/rstudio/shared
                         subPath: _shared
                         readOnly: true
+                    init_containers:
+                      # Need to explicitly fix ownership here, as otherwise these directories will be owned
+                      # by root on most NFS filesystems - neither EFS nor Google Filestore support anonuid
+                      - name: volume-mount-ownership-fix
+                        image: busybox
+                        command:
+                          [
+                              "sh",
+                              "-c",
+                              "id && chown 1000:1000 /home/rstudio && chown 1000:1000 && ls -lhd /home/rstudio ",
+                          ]
```

Comment:
```suggestion
                          - sh
                          - -c
                          - id && chown 1000:1000 /home/rstudio && ls -lhd /home/rstudio
```

Reply:
thanks @consideRatio! yeah, saw this last night but didn't have time to do it until this morning. Adding fix now